calico_node/startup/startup.go

```diff
@@ -341,6 +341,22 @@ func validateIP(ipn *net.IPNet) {
 	warning("Unable to confirm IPv%d address %s is assigned to this host", ip.Version(), ip)
 }
 
+// evaluateENVBool evaluates a passed environment variable
+// Returns an boolean indicating if the parameter was defined
+// and an boolean if the argument was true or false
+// If it was false an additional log entry will be written
+
+func evaluateENVBool(envvar string, logentry string) (isDefined bool, outgoingNATEnabled bool) {
+	envValue, isEmpty := os.LookupEnv(envvar)
+	switch strings.ToLower(envValue) {
+	case "false", "0", "no", "n", "f":
+		log.Info(logentry)
+		return isEmpty, false
+	}
+
+	return isEmpty, true
```

Comment:
> do you think the isDefined return value should remain? I don't think so.

Yeah, I missed that - we don't need the isDefined return value if we take this approach.  If we need an isDefined down the road for some other bit of code, we can add that (or that code can check it explicitly).